Update tickets_list.php

tysoe27 opened the pull request:
Checks to see if status is set in the GET paramaters. Without this, a warning is shown (if PHP errors are enabled):

**Warning:** Undefined array key "status" in **tickets_list.php** on line **12**

tysoe27 committed:
Update tickets_list.php

Checks to see if status is set in the GET paramaters. Without this, a warning is shown (if PHP errors are enabled):

Warning: Undefined array key "status" in tickets_list.php on line 12

johnnyq commented:
nice catch before I can merge a few suggestions you have a double } at the end and should use && instead of another of another if condition

tysoe27 committed:
Update tickets_list.php

Remove the check for "Closed" status to prevent a Warning when no statuses are selected. Otherwise the warning that shows is:

Warning: Undefined array key "status" in tickets_list.php on line 12
